### DISCUSSION TITLE
Editing Jasper Files

### DISCUSSION THREAD

**Discussion by allprecisionmfgpcode** [blueseerERP/blueseer#84] [Q&A] [answered]
I am attempting to edit the Jasper Files to customize the reports how I need them, but once I customize the report it will no longer function from within the ERP system. Even if I open just the jrxml file and post it without making changes it no longer works. Am I missing a step that i need to follow in the JasperSoft Studio application?

**Comment by vaughnte** [comment, marked as answer]
In the JasperSoft Studio, make sure you 'compile' the .jrxml file into a .jasper file.   The .jasper file is the file that gets executed by the BlueSeer app.   If you did attempt to compile, then make sure there are no run time errors from your new code changes.   Look in the root directory for the data/app.log file.   If it exists...then there may be an indication of a run-time error.   Feel free to send it to support@blueseer.com as necessary.   Also, it's good practice to backup your jasper/xxxxx.jasper compiled code before compiling the new .jasper....it will overwrite the old...and you may need to restore it.    If necessary...you can get all the .jasper (and .jrxml) files from this github site and replace into the jasper/ directory as required.

**Reply by vaughnte** [reply]
note...all jasper files in this github repo are under the 'sf/jasper' directory....in case you need to recover.
